Monotype/c2pa-rs
# C2PA-557: Update c2pa-rs fontSupport to pull in main

## Changes in this pull request

Pull in most recent `main` to catch our `monotype/fontSupport` branch up.

## Verification Instructions

- Pull this repo's `feature/C2PA-557/updateC2paRs` branch
- Pull https://github.com/Monotype/c2patool with the `monotype/fontSupportLocal` branch
- Build `c2patool`
  - `cargo build`
- Use `c2patool` to sign a font
  -  `.\target\debug\c2patool.exe -m .\sample\test.json "CultStd.otf" -f -d -o "CultStd_signed.otf"`
- Inspect the font with c2patool to verify that it was correctly signed and that all signature/hashes are valid
  - `.\target\debug\c2patool.exe -d "CultStd_signed.otf"`

## Checklist
- [ ] This PR represents a single feature, fix, or change.
- [ ] All applicable changes have been documented.
- [ ] Any `TO DO` items (or similar) have been entered as GitHub issues and the link to that issue has been included in a comment.
